### ISSUE TITLE
Cannot toggle dropdowns using Enter key

### ISSUE THREAD

**Issue by sorin-davidoi** [tootsuite/mastodon#5382]
Dropdowns can only be toggled using Space. Also, when toggling the dropdowns, the focus is not moved inside, so the keyboard cannot be used to select any of the options.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by cacheflow** [comment]
Could you upload gif/video of this or steps to reproduce?

**Comment by sorin-davidoi** [comment]
Move the focus to the privacy drop down and press Enter - it should open it and you should be able to use the arrow keys to select an option.

**Comment by mattcoxonline** [comment]
Attaching a GIF of not being able to tab into the search or elipses dropdown menu. I've made the focus state a red border so you can see where I'm tabbing.

![nov-15-2017 14-34-57](https://user-images.githubusercontent.com/673542/32838755-5d5fa4c4-ca12-11e7-93f3-59f07214d1fb.gif)

![nov-15-2017 14-36-13](https://user-images.githubusercontent.com/673542/32838756-5d7ccc20-ca12-11e7-8883-e1c8ae7a1fab.gif)

**Comment by MarcoZehe** [comment]
I can confirm this problem.

**Comment by Gargron** [comment]
Fixed by #7318
